### ISSUE TITLE
Cannot use [N]byte in Query.Where()

### ISSUE THREAD

**Issue by katomaso** [go-pg/pg#1521]
Maybe I am doing something wrong. Could you please advice
```go
type Person struct {
	Email     string `pg:",pk"`
	Password  [sha512.Size]byte
}

func Authenticate(email, password string) (*Person, error) {
	var (
		person = &Person{Email: email}
	)
	person.SetPassword(password) // simply uses sha512.Sum512([]byte(password))
	exists, err := backend.Db().Model(person).
		Where("email=?", person.Email).
		Where("password=?", person.Password). // this line causes the panic
		Exists()
}
```

Function Authenticate panics on

```
panic serving 127.0.0.1:49516: reflect.Value.Slice: slice of unaddressable array
...
github.com/go-pg/pg/v9/types.appendArrayBytesValue(0xc000204000, 0x4c, 0x10000, 0x14dd2c0, 0xc0000a7fc0, 0x91, 0x1, 0x100be53, 0x1507460, 0x14dd2c0)
	/Users/almo/go/pkg/mod/github.com/go-pg/pg/v9@v9.1.2/types/append_value.go:168 +0x84
github.com/go-pg/pg/v9/types.appendValue(0xc000204000, 0x4c, 0x10000, 0x14dd2c0, 0xc0000a7fc0, 0x91, 0x1, 0x3c, 0xc000444d20, 0x100be53)
	/Users/almo/go/pkg/mod/github.com/go-pg/pg/v9@v9.1.2/types/append_value.go:136 +0x19d
```

I am using Postgres, column `password` is created as type `bytea`. Inserting works, only searching (using mentioned `Where`) panics.

**Comment by vmihailenco** [comment]
Thanks for the report - should be fixed in v9.1.3

**Comment by katomaso** [comment]
Thanks for the fix :)
